### ISSUE TITLE
SCT support for muon intensity fitting

### ISSUE THREAD

**Issue by MaxNoe** [cta-observatory/ctapipe#1259]
It's currently not supported at all, but the functions interface implies so.

There is even code like this:
```python
if self.sct_flag:
    self.sct_flag = False  # Do not treat differently for now...work in progress
````

**Comment by kosack** [comment]
Is this for all SC telescopes? Or do you mean the pSCT?

**Comment by MaxNoe** [comment]
Any telescope not having just one primary mirror

**Comment by kosack** [comment]
Maybe just raise a `NotImplementedError`, so we don't forget

**Comment by kosack** [comment]
Certainly there should not be a configurable `sct_flag` as that can be read from the `TelescopeDescription`.

**Comment by MaxNoe** [comment]
Yes, I'm current rewring the whole muon stuff to properly use telescope descriptions

**Comment by maxnoe** [comment]
Reference to the paper by Markus: https://iopscience.iop.org/article/10.3847/1538-4365/ae6da5/meta
